public/locales/de/translation.json

```diff
@@ -256,6 +256,16 @@
     "colorSchemeLight": "Hell",
     "colorSchemeDark": "Dunkel"
   },
+  "Feedback": {
+    "FeedbackInputLabel": "Möchtest du uns noch etwas mitteilen?",
+    "FeedbackInputPlaceholder": "Hier kannst du deine Gedanken festhalten!",
+    "ContactInputLabel": "Wenn du möchtest, dass wir dich kontaktieren, hinterlasse deine E-Mail!",
```

Comment:
I omitted it because the german language is noticeable longer than it's English equivalent and adding `(Optional)` or `(Freiwillig)` to the text would cause a line break into a third line on certain screen widths

Reply:
I think a good solution would be to remove `(Optional)` from the english translation since the text clearly hints that it isn't mandatory with `If you would like us...`